libsolidity/analysis/ConstantEvaluator.cpp

```diff
@@ -407,3 +409,24 @@ void ConstantEvaluator::endVisit(TupleExpression const& _tuple)
 	if (!_tuple.isInlineArray() && _tuple.components().size() == 1)
 		m_values[&_tuple] = evaluate(*_tuple.components().front());
 }
+
+void ConstantEvaluator::endVisit(MemberAccess const& _memberAcess)
```

Comment:
Sorry I just see this now, there's a typo here...

```suggestion
void ConstantEvaluator::endVisit(MemberAccess const& _memberAccess)
```